src/test/java/org/dependencytrack/resources/v1/FindingResourceTest.java

```diff
@@ -688,174 +690,135 @@ public void getSARIFFindingsByProjectTest() {
         Assert.assertEquals(200, response.getStatus(), 0);
         Assert.assertEquals(MEDIA_TYPE_SARIF_JSON, response.getHeaderString(HttpHeaders.CONTENT_TYPE));
         final String jsonResponse = getPlainTextBody(response);
+        JSONArray resultArray = new JSONObject(jsonResponse).getJSONArray("runs").getJSONObject(0).getJSONArray("results");
 
         assertThatJson(jsonResponse)
                 .withMatcher("version", equalTo(new About().getVersion()))
-                .withMatcher("fullName", equalTo("OWASP Dependency-Track - " + new About().getVersion()))
```

Comment:
You can achieve the same outcome without the need to parse intermediary objects with:

https://github.com/DependencyTrack/hyades-apiserver/blob/a48b7973514b9c07bf6a84de24651fac8e395dce/src/test/java/org/dependencytrack/model/mapping/PolicyProtoMapperTest.java#L79